makefiles/Makefile.devhub

```diff
@@ -27,15 +27,15 @@ include ~/shared.mk
 
 update-snooty:
 	npm install --production > /dev/null 2>&1
-	
+
 next-gen-html: update-snooty
 	# get latest devhub-content to test the front-end
 	rm -rf devhub-content && \
 	git clone https://${GITHUB_USER}:${GITHUB_PASS}@github.com/10gen/devhub-content devhub-content && \
 	cp devhub-content/snooty.toml snooty.toml && \
 	cp -r devhub-content/source source; \
 	# snooty parse and then build-front-end
-	@-echo ${SNOOTY_DB_PWD} | snooty build ${REPO_DIR} "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true" --commit "${COMMIT_HASH}" || exit 0;
+	@-echo ${SNOOTY_DB_PWD} | snooty build ${REPO_DIR} "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true" --commit "${COMMIT_HASH}" ${RSTSPEC_FLAG} || exit 0;
```

Comment:
Do we need to update the devhub makefile?

Reply:
Is there a reason not to? Otherwise, the content on devhub will have links to the old content, right?

Reply:
Reverted the devhub changes!